### PR TITLE
Convert tables in moddb database to the InnoDB table engine

### PR DESCRIPTION
**Pull request by eiffel777**
Convert tables in moddb database to the InnoDB table engine

## Tests performed
Tested in docker

## Checklist:

- [x] The pull request description is suitable for a Changelog entry
- [x] The milestone is set correctly on the pull request
- [x] The appropriate labels have been added to the pull request

### CODE REVIEW

**Comment by ryanrath** [comment]
Just a quick question ( that might have already been accounted for ), are the rest of the tables in `moddb` going to be migrated as well / have they already been migrated?

**Comment by eiffel777** [comment]
@ryanrath All the other tables in `moddb` already use the InnoDB table engine.

**Comment by ryanrath** [comment]
Sweet, figured it was something like that :) Thanks!

File: tests/ui/webdriverHelpers/waitForInvisible.js [hunk 1]
```diff
@@ -8,6 +8,6 @@
  */
 
 module.exports = function waitForInvisible(selector, ms) {
-    var timeOut = ms || 5000;
+    var timeOut = ms || 9000;
```
Comment (jpwhite4) [inline tests/ui/webdriverHelpers/waitForInvisible.js:11]: Is this change supposed to be in the pull request?

Comment (eiffel777) [reply tests/ui/webdriverHelpers/waitForInvisible.js]: @jpwhite4 Nope. I added that in as a test when I was having trouble getting the UI tests to pass. It should be removed now.

**Pull request closed without merging**